### ISSUE TITLE
Admin\Operations - Internal User can view Operations

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1701]
#### Description:
As an IRC user, given that I have access, I can view all the operations from the industry.
- [Design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7861-688&t=pmKjK1RmQeLYOb04-4)
- [Prototype](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7781%3A6398&type=design&node-id=7861-688&viewport=74%2C199%2C0.46&t=xuYjeIKfjwjoCJNb-1&scaling=scale-down&starting-point-node-id=7781%3A6471)

#### Acceptance Criteria:
Given I am an IRC user
When I click on the Operations sub link from the [BCIERS Dashboard](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7781%3A6398&type=design&node-id=7781-6471&viewport=74%2C199%2C0.46&t=xuYjeIKfjwjoCJNb-1&scaling=scale-down&starting-point-node-id=7781%3A6471) or the Operations tile from the [Administration Dashboard](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7781%3A6398&type=design&node-id=7781-6399&viewport=74%2C199%2C0.46&t=xuYjeIKfjwjoCJNb-1&scaling=scale-down&starting-point-node-id=7781%3A6471),
Then I am brought to the Operations page with the Operations grid as in the design
Then I can sort/search by columns

When I click on the View Details link of an operation
Then I'm brought to the Operation Details page of the Operation as in the prototype

#### Development Checklist:

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies
Blocked by #1686

**Comment by nanyangpro** [comment]
Design/Prototype links are added or updated with AC refined further.

**Comment by BCerki** [comment]
This will likely be completed as part of the external component ticket: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=64781153. We're using the same component for both with some conditional logic, and most of the part II are the same for both grids.

**Comment by shon-button** [comment]
@patriciarussellCAS

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/operations

**Comment by patriciarussellCAS** [comment]
Reviewed in Dev.
